### ISSUE TITLE
Deploying with Rasa doesn't work.

### ISSUE THREAD

**Issue by timlukins** [aws/chalice#720]
Hi, 

Really liking chalice - but was trying something the other day that seems to flummox my local test deployment to a folder.

As title says: trying to bundle this with Rasa (http://rasa.com).

But including Rasa seems to result in a missing output file/folder for deployment.

Isolating the issue is possible by doing the following:

```
tim$ chalice --version
chalice 1.1.0

tim$ chalice new-project testrasa

tim$ cd testrasa

tim$ echo "rasa_core" >> requirements.txt 

tim$ chalice package deploy
```

[Time passes....]

```
Creating deployment package.
/var/folders/mn/z19ts48937n8tb2v3n6hl5xc0000gn/T/tmptvnugz3f/Twisted-17.9.0.tar.bz2

tim$ ls deploy
ls: deploy: No such file or directory
```

Similar thing occurs if I use `--single_file` option...

(Might perhaps be something to do with the sheer size of this?)

Thanks. T

**Comment by stealthycoin** [comment]
Looking at Rasa it has 116M of just pure dependencies. Lambda has a package size limit of 50MB [currently](https://docs.aws.amazon.com/lambda/latest/dg/limits.html) so rasa won't be deployable to Lambda regardless of Chalice packaging it or not.

That being said it does appear that there is an issue with packaging Twisted here. The extension is `tar.bz2` which is technically not a valid packaging extension given PEP 527, though it can be made to work. So I will mark this as a feature request specifically for `bz2` support for packaging in general, rasa still won't work though since its far too large of a dependency tree.

**Comment by timlukins** [comment]
Yep - I thought it might have been a size issue. Thanks for confirming. It was just an idea to try this out, but obviously Rasa is probably better deployed as it's own service - as it's not exactly "micro".
